runastyle

```diff
@@ -76,3 +84,9 @@ done
 
 formatXML man/cppcheck.1.xml
 formatXML cppcheck-errors.rng
+
+# If not available get autopep8 from pip
+if ! command -v $1 ; then
+  pip install --user --upgrade autopep8
```

Comment:
was this by intention? on my computer `pip` is for python 2.x.. and `pip3` is for python 3.x. Would the python 3.x autopep8 tool have problems with our python 2.x code?